src/herder/test/UpgradesTests.cpp

```diff
@@ -2025,6 +2040,30 @@ TEST_CASE("configuration initialized in version upgrade", "[upgrades]")
             CONFIG_SETTING_CONTRACT_MAX_SIZE_BYTES);
     REQUIRE(maxContractSizeEntry.contractMaxSizeBytes() ==
             InitialSorobanNetworkConfig::MAX_CONTRACT_SIZE);
+
+    // Check that BucketList size window initialized with current BL size
+    REQUIRE(bm.getAverageBucketListSize(ltx) == blSize);
```

Comment:
We'll also need some tests to cover upgrades involving increasing/decreasing the window size. It's ok to do that as a followup though.